include/deal.II/base/parsed_convergence_table_flags.h

```diff
@@ -0,0 +1,232 @@
+//-----------------------------------------------------------
+//
+//    Copyright (C) 2019 by the deal.II authors
+//
+//    This file is part of the deal.II library.
+//
+//    The deal2lkit library is free software; you can use it, redistribute
```

Comment:
I realized that this change is not enough to get the standard header formatting so I went ahead and pushed two commits to do this instead of putting in suggested changes.